back/app/mailers/confirmations_mailer.rb

```diff
@@ -11,6 +11,11 @@ def send_confirmation_code
     end
   end
 
+  def to_email
+    email = recipient.new_email.presence || recipient.email
+    email_address_with_name(email, "#{recipient.first_name} #{recipient.last_name}")
```

Comment:
I wonder if we should use the UserDisplayNameService here, to deal with abbreviated names, missing names etc?

Reply:
I don't think this is needed here as it's only for sending an email to the user and seems to work as it is even if there are missing names